src/ios/ZeroConf.swift

```diff
@@ -436,4 +433,36 @@ import Foundation
         return nil
     }
 
+    fileprivate static func dictionary(fromTXTRecord txtData: Data) -> [String: String] {
+
+        var result = [String: String]()
+        var data = txtData
+
+        while !data.isEmpty {
+            // The first byte of each record is its length, so prefix that much data
+            let recordLength = Int(data.removeFirst())
```

Comment:
This is probably covered by the spec or data structure, but just in case: it's not possible for the length to be greater than one byte, right?